app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if command.split()[0] == "mv" and command.split() == 3:
+        command, source_file_name, new_file_info = command.split()
+        if "/" not in new_file_info:
+            new_file_name = new_file_info
+        else:
+            directories, file_name = os.path.split(new_file_info)
+            if directories:
+                os.makedirs(directories, exist_ok=True)
+            new_file_name = os.path.join(directories, file_name)
```

Comment:
You can remove if-else here
```suggestion
        directories, file_name = os.path.split(new_file_info)
        if directories:
            os.makedirs(directories, exist_ok=True)
        new_file_name = new_file_info # attention here you can remove this line
```